Research/Research/RSDSchedule.swift

```diff
@@ -37,6 +37,9 @@ public protocol RSDScheduleTime {
     
     /// The time of the day as a string with the format "HH:mm:ss" or "HH:mm".
     var timeOfDayString: String? { get }
+    
+    /// The time zone for the time of day.
```

Comment:
```suggestion
    /// The original time zone for the time of day.
```
(to clarify the intended usage/purpose)?